docs/Unity-Inference-Engine.md

```diff
@@ -27,10 +27,17 @@ but we only tested for the following platforms :
 * iOS
 * Android
 
+## Supported formats
+There are currently two supported model formats:
```

Comment:
@surfnerd @ervteng - I made this change to the docs too. Can you take a look when you get a chance? (I might merge first and revise if needed).